host/crypto/bcrypt/rsa.c

```diff
@@ -175,6 +175,19 @@ void oe_rsa_public_key_init(
         (oe_bcrypt_key_t*)public_key, key_handle, OE_RSA_PUBLIC_KEY_MAGIC);
 }
 
+
+oe_result_t oe_rsa_private_key_from_engine(
+    oe_rsa_private_key_t* private_key,
+    const char *engine_id,
+    const char *engine_load_path,
+    const char *key_id)
+{
+    /*
+     * bcrypt does not support engines, so nothing to do. 
```

Comment:
should "engines" be "plugins"  (this goes back to the terminology question of whether it's 1 engine with multiple plugins, or multiple engines)

Reply:
plugin is not the correct term. If we say plugin it will not correspond to the usage in openssl and would tend to confuse the target audience.  The doc on openssl engines is already garbled enough, so varying terminology is not good.